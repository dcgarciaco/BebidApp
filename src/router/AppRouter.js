import React from 'react';

import {
  BrowserRouter as Router,
  Switch,
  Route,
  Redirect,
  //   Link
} from 'react-router-dom';
import { Navbar } from '../components/ui/Navbar';

import { Beers } from '../components/beers/Beers';
import { Cocktails } from '../components/cocktails/Cocktails';
import { OrdinaryDrinks } from '../components/ordinaryDrink/OrdinaryDrinks';
import { DrinkItem } from '../components/drinks/DrinkItem';

export const AppRouter = () => {
  return (
    <Router>
      <header>
        <Navbar />
      </header>
      <article>
        <Switch>
          <Route exact path='/' component={Cocktails} />
          <Route exact path='/cocktails' component={Cocktails} />
          <Route exact path='/ordinary_drinks' component={OrdinaryDrinks} />
          <Route exact path='/beers' component={Beers} />
          <Route exact path='/drink/:drinkID' component={DrinkItem} />

          <Redirect to='/' />
        </Switch>
      </article>
    </Router>
  );
};
