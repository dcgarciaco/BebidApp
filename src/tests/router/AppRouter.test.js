import { shallow } from 'enzyme';
import { AppRouter } from '../../router/AppRouter';

describe('Test in <AppRouter />', () => {
  test('should shows correctly', () => {
    const wrapper = shallow(<AppRouter />);
    expect(wrapper).toMatchSnapshot();
  });
});
