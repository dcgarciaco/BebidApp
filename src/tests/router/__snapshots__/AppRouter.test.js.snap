// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test in <AppRouter /> should shows correctly 1`] = `
<BrowserRouter>
  <header>
    <Navbar />
  </header>
  <main>
    <Switch>
      <Route
        component={[Function]}
        exact={true}
        path="/"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/cocktails"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/ordinary_drinks"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/beers"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/drink/:drinkID"
      />
      <Redirect
        to="/"
      />
    </Switch>
  </main>
</BrowserRouter>
`;
